doc/user_guide/psyir.rst

```diff
@@ -665,3 +665,35 @@ parent reference are not.
 .. code-block:: python
 
     new_node = node.copy()
+
+Named arguments
+---------------
+
+The `Call` and three sub-classes of `Operation` node
+(`UnaryOperation`, `BinaryOperation` and `NaryOperation`) all support
+named arguments.
+
+Named arguments can be set or modified via the `create()`,
+`append_named_arg()`, `insert_named_arg()` or `replace_named_arg()`
+methods.
+
+If an argument is inserted directly (via the children list) then it is
+assumed that this is not a named argument. If (the parent node of) an
```

Comment:
"the top node" ? Given that the parent has another meaning in PSyIR nodes

Reply:
Yes, good point.